src/openrct2/world/Footpath.cpp

```diff
@@ -1426,8 +1426,7 @@ static int32_t footpath_is_connected_to_map_edge_helper(CoordsXYZ footpathPos, i
                         continue;
```

Comment:
L1417 isn't checking if the tileElement is the last element either. There are a few odd/wrong things with this.

Reply:
Hm right it should definitely start at the first tile, it probably started at 1 to always skip the surface.

Reply:
No sorry it's meant to start at 1 it's just it's meant to check 0 to make sure it's not the last. At this point in the function we already point at a path. What this loop is meant to do is find all the banners on that path.